GameData/RealismOverhaul/RO_SuggestedMods/Squad/RO_Squad_Engines.cfg

```diff
@@ -1047,7 +1047,39 @@
 	}
 }
 
-@PART[sepMotor1|ROSmallSpinMotor|SnubOtron|sepMotorLarge]:FOR[RealismOverhaul]
+//  ==================================================
+//  Radial Separation motor (small).
+//  ==================================================
+
++PART[sepMotor1]:AFTER[RealismOverhaul]
+{
+	@name = sepMotorSmall
+	@title = Radial Separation Motor (Small)
+	@description = Small solid motor use to help separate one stage from another or perform ullage. Best used with others.
+
+	%rescaleFactor = 0.5
+	@mass = 0.008
+
+	@MODULE[ModuleEngines*]
+	{
+		@maxThrust *= 0.18
+	}
```

Comment:
I think this is unnecessary since they have an MEC.  (And could be removed from the SepMotorLarge copy, and also from the SepMotor1 original patch.)  The advantage of MEC is we don't need to manipulate the ME/MERF directly.

Reply:
Done in https://github.com/KSP-RO/RealismOverhaul/pull/2454/commits/da7dfdb705a7bf91b7ee135d81c968ae24c176bf